schedule/RAID1_staging.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name:           RAID1
+description:    >
+  Test for RAID1.
+  Installation of RAID1 using expert partitioner.
+vars:
+  RAIDLEVEL: 1
+conditional_schedule:
+  sle15_workarounds:
+    VERSION:
+      15-SP1:
+        - console/sle15_workarounds
+  sle15accept_licence:
+    VERSION:
+      15-SP1:
```

Comment:
Hmm, I somehow missed that we used this variable. We should not use `VERSION` or at least not matching exact value. So we need to understand why we don't have it for SLE 12 and potentially create separate schedule for it. 
We cannot accept this solution because now if SP2 is out there we have to change all relevant schedules.
If this cannot be avoided, we can implement special `VERSION` checks to allow ranges.